anchor-service/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+mod anchor_batch;
```

Comment:
Can you add `#![warn(missing_docs)]` here so we document all public interfaces from this crate.